### DISCUSSION TITLE
Brain Extraction

### DISCUSSION THREAD

**Discussion by Meddebma** [Project-MONAI/MONAI#3085] [Ideas]
Dear MONAI Team, 

I would like to start a project of segmentation and classification of Brain lesions using CT-Images. I was wondering if you have experience with brain extraction and skull stripping. There are few open source methods like CT-Seg (https://github.com/WCHN/CTseg) but it is only on MATLAB, It would be nice to make a python code for it. 

Thanks!

**Comment by Nic-Ma** [comment]
Hi @ericspod @rijobro ,

Could you please help share some comments if you have any related experience?
Is it useful to refer to our brain tumor segmentation tutorial?
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/brats_segmentation_3d.ipynb

Thanks in advance.

**Comment by function2-llx** [comment]
I would recommend [HD-BET](https://github.com/MIC-DKFZ/HD-BET/), an out-of-box brain extraction tool with excellent performance.

**Reply by inikishev** [reply]
Hi @function2-llx ! Is there a version of HD-BET for CT images?
